understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
+                     <li>非テキストコントラストでは、フォーカスインジケータは、インジケータ (フォーカス状態) と隣り合う非インジケータ色との間に<em>隣接コントラスト</em>を持つことが必要である。
                      </li>
                      
                   </ul>
                   
                   
                   <figure id="non-text-contrast-fail-focus-appearance-pass">
-                     <img src="img/focus-indicator-good-change-bad-adjacent.png" alt="A light blue button with a 3px thick border. The border is black in the unfocused state and light gray in the focused state." />
+                     <img src="img/focus-indicator-good-change-bad-adjacent.png" alt="太さ 3px のボーターを持つ水色のボタン。ボーターは、フォーカスされていない状態では黒で、フォーカスされている状態では薄いグレーになる。" />
                      
-                     <figcaption>Figure 18 This example passes Focus Appearance but fails Non-text Contrast; there is insufficient
-                        <em>adjacent contrast</em> between the focus indicator and the adjacent colors.
+                     <figcaption>図 18 この例は、フォーカスの外観は合格しているが、非テキストのコントラストは不合格である。フォーカスインジケータと隣接する色の間の<em>隣接コントラスト </em>が不十分である。
                      </figcaption>
                      
                   </figure>
                   
                   
                   <figure id="non-text-contrast-pass-focus-appearance-fail">
-                     <img src="img/focus-indicator-good-adjacent-bad-change.png" alt="A light blue button with a 3px thick border. The border is black in the unfocused state and dark gray in the focused state." />
+                     <img src="img/focus-indicator-good-adjacent-bad-change.png" alt="太さ 3px のボーターを持つ水色のボタン。ボーターは、フォーカスされていない状態では黒で、フォーカスされている状態では濃いグレーになる。" />
                      
-                     <figcaption>Figure 19 This example passes Non-text Contrast but fails Focus Appearance; there is insufficient
-                        <em>change of contrast</em> between the focused and unfocused states.
+                     <figcaption>図 19 この例は、非テキストコントラストは合格している、フォーカスの外観は不合格である。フォーカスされた状態とフォーカスされていない状態との間の<em>コントラストの変化</em>が不十分である。
                      </figcaption>
                      
                   </figure>
                   
                   
-                  <p>Additionally, Non-text Contrast does not establish any size requirement and has slightly
-                     different rules for when exceptions are allowed.
+                  <p>さらに、非テキストのコントラストではサイズ要件が設けられておらず、例外が認められる場合のルールも若干異なる。
                   </p>
                   
                   
-                  <p>See the Relationship with Focus Visible section of <a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">Understanding 1.4.11 Non-text Contrast</a> for more details and examples.
+                  <p>詳細と例については<a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">解説書 1.4.11 非テキストのコントラスト</a>のフォーカスの可視化との関係の節を参照。
```

Comment:
「フォーカスの可視化との関係の節」ですが、WCAG 2.1 解説書の該当箇所を見ると、「フォーカスの可視化との関係**性**」と訳されているので、それに合わせるのがよいと思います。
https://waic.jp/translations/WCAG21/Understanding/non-text-contrast.html

```suggestion
                  <p>詳細と例については<a href="https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html">解説書 1.4.11 非テキストのコントラスト</a>のフォーカスの可視化との関係性の節を参照。
```